packages/components/src/components/codeChunk/codeChunk.tsx

```diff
@@ -23,6 +23,11 @@ interface CollapseEvent extends CustomEvent {
   scoped: true
 })
 export class CodeChunk {
+  public static slots = {
+    text: 'text',
+    output: 'output'
```

Comment:
To be consistent with https://schema.stenci.la/CodeChunk.schema.json this should be `outputs`. There are other properties that we should probably add sooner, rather than later, because they will be able to be populated from Jupyter Notebooks and elsewhere: `programmingLanguage` and `errors`.